charts/prefect-server/templates/deployment.yaml

```diff
@@ -141,6 +143,9 @@ spec:
             - mountPath: /tmp
               name: scratch
               subPathExpr: tmp
+            - mountPath: {{ .Values.server.uiConfig.prefectUiStaticDirectory }}
+              name: scratch
```

Comment:
nothing really special about this directory, so we can re-use the emptyDir that we initialize here